deploy/olm-catalog/opendatahub/0.8.0/opendatahub-operator.v0.8.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,364 @@
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "airflow/operator"
+                  }
+                },
+                "name": "airflow-operator"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "airflow/cluster"
+                  }
+                },
+                "name": "airflow-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odhargo/cluster"
+                  }
+                },
+                "name": "odhargo-cluster"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odhargo/odhargo"
+                  }
+                },
+                "name": "odhargo"
+              }
+            ],
+            "repos": [
+              {
+                "name": "kf-manifests",
+                "uri": "https://github.com/opendatahub-io/manifests/tarball/v1.0-branch-openshift"
+              },
+              {
+                "name": "manifests",
+                "uri": "https://github.com/opendatahub-io/odh-manifests/tarball/v0.8.0"
+              }
+            ],
+            "version": "v0.8.0"
+          },
+          "status": {}
+        }
+      ]
+    capabilities: Basic Install
+    categories: AI/Machine Learning, Big Data
+    description: "Open Data Hub Operator for deployment and management of Open Data Hub and Kubeflow on OpenShift"
+    support: Open Data Hub
+    repository: https://github.com/opendatahub-io/opendatahub-operator
+    createdAt: '2020-08-31T00:00:00Z'
+    containerImage: quay.io/opendatahub/opendatahub-operator:v0.8.0
+    certified: 'False'
+  name: opendatahub-operator.v0.8.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: KfDef is the Schema for the applications API
+      kind: KfDef
+      name: kfdefs.kfdef.apps.kubeflow.org
+      version: v1
+      displayName: Open Data Hub
+      group: kfdef.apps.kubeflow.org
+  description: |
+    The Open Data Hub is a machine-learning-as-a-service platform built on Red Hat's Kubernetes-based OpenShift® Container Platform, Ceph Object Storage, and integrating a collection of open source projects.
+
+    Open Data Hub is a meta-project that integrates open source projects into a practical solution. It aims to foster collaboration between communities, vendors, user-enterprises, and academics following open source best practices. The open source community can experiment and develop intelligent applications without incurring high costs and having to master the complexity of modern machine learning and artificial intelligence software stacks.
+
```

Comment:
```suggestion

    Open Data Hub operator allows users to install and manage Open Data Hub and Kubeflow components. Users can mix and match tools from each project to fulfill the needs of their use case. Each of the projects share some components, but canbe mostly seens as an extension of each other to provide as complete solution as possible for both hobby and enterprise users.
```

Reply:
How about adding something like this ^ @crobby @nakfour @LaVLaS ?

Reply:
Typo fix:

`...but can be mostly seen as an extension of each other to provide as complete a solution as possible for both hobby and enterprise users.`

Reply:
Updated the suggestion

Reply:
Updated